src/main/java/functional/Parser.java

```diff
@@ -0,0 +1,25 @@
+package functional;
+
+import commands.*;
+
+public class Parser {
+
+    public static Command parse(String fullCommand) {
+        String[] input = fullCommand.split(" ");
+        switch (input[0]) {
+            case "bye":
+                return new ExitCommand();
+            case "delete":
+                return new DeleteCommand();
+            case "list":
+                return new ListCommand();
+            case "mark":
+                return new MarkCommand();
+            case "unmark":
+                return new UnmarkCommand();
+            default:
+                return new AddCommand();
```

Comment:
Perhaps no indentation for case clauses would be better as it is part of the coding standard?